spread.yaml

```diff
@@ -176,7 +176,7 @@ backends:
             - ubuntu-20.04-64:
                   image: ubuntu-2004-64-virt-enabled
                   storage: 20G
-                  workers: 6
+                  workers: 7
```

Comment:
While I agree with the change here it looks a bit unrelated. It's a separate commit so probably ok